README.md

```diff
@@ -12,21 +12,44 @@ https://github.com/user-attachments/assets/85b669af-e4c6-4fe6-9ad3-b5cce26b1a08
 
 ## Running locally
 
-First generate `auth.conf` for the NATS cluster and various `.creds` and `.nk` files for NATS authentication:
+### Prerequisites
+- **Docker Desktop** and **Podman Desktop** (recommended for easiest setup)
+- **NATS Tools**: Command-line utilities for interacting with [NATS](https://github.com/nats-io/natscli?tab=readme-ov-file#installation)
+
+### Configuration
+
+1. **Set up Environment Variables**  
+Add Github and personal token information in .env file.
+
+```makefile
+GITHUB_USERNAME=your_github_username
+GITHUB_TOKEN=your_personal_token
+```
+
+2. **Generate NATS Authentication Files**  
```

Comment:
In markdown, you should be able to change this to `1.`. That way, if we insert an extra step we don't have to redo the count below it.